### DISCUSSION TITLE
How can I look up the word in the main window instead of pop up window when I execute `goldendict word`?

### DISCUSSION THREAD

**Discussion by otakutyrant** [xiaoyifang/goldendict-ng#1072] [Q&A] [answered]
It worked as I expected. But after reinstalling Linux system and goldendict-ng, it suddenly shows result in a pop window.

**Comment by shenlebantongying** [comment, marked as answer]
Updated new method: https://github.com/xiaoyifang/goldendict-ng/pull/1173#issue-1914456812 (`goldendict -s <word>` will always send word to the popup)

---

You can enable this option ![image](https://github.com/xiaoyifang/goldendict-ng/assets/20123683/abbb86ad-9958-4023-b1d4-ca341137b471)

**Reply by otakutyrant** [reply]
Thank you.

**Reply by KonstantinDjairo** [reply]
it doesn't work on freebsd when reading the clipboard, it just doesn't show anything, only works in popup mode

**Comment by xiaoyifang** [comment]
maybe implement a new feature to specify which windows to show the translation entry.
such as 
`goldendict --main word`
or 
`goldendict --popup word`
or 
`goldendict --windows=main|popup word`

**Reply by otakutyrant** [reply]
I advocate that. If goldendict has a good command interface, it is beneficial for script programming or interaction between other command utils.

**Comment by KonstantinDjairo** [comment]
it would be desirable to also ensure that it sends the result to the main window, or just create a function that doesn't use that popup window at all.
i tried to use the clipboard track feature in freebsd but it doesn't seem to work properly.
another option would be letting the user specify a custom clipboard program, such as xclip or sselp

**Comment by xiaoyifang** [comment]
https://github.com/xiaoyifang/goldendict-ng/pull/1173
